openmetadata-ui/src/main/resources/ui/src/components/NotificationBox/NotificationBox.component.tsx

```diff
@@ -115,7 +115,7 @@ const NotificationBox = ({
           key === NotificationTabsKey.TASK
             ? onMarkTaskNotificationRead()
             : onMarkMentionsNotificationRead();
-        }, 4000);
+        }, 2500);
```

Comment:
can we make const and use everywhere as it will help in future if need change the time